src/database/migrations/20230609052656-create-user-extension.js

```diff
@@ -23,6 +23,9 @@ module.exports = {
 			meta: {
 				type: Sequelize.JSONB,
 			},
+			status: {
```

Comment:
can refer this and add new migration for status https://sequelize.org/docs/v6/other-topics/query-interface/ @adithyadinesh0412